src/types.ts

```diff
@@ -32,8 +33,20 @@ export interface ZAuctionInstances {
   [registrarAddress: string]: zAuction.Instance;
 }
 
-export interface RouteUriToInstance {
-  [key: string]: ZAuctionInstances;
+export interface TokenAddressMapping {
+  [address: string]: Mapping;
+}
+
+export interface Mapping {
+  [address: string]: {
+    id: string,
```

Comment:
Make this a separate interface instead of inline. This should also be `Maybe<T>`